airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -17,4 +17,20 @@ $(document).ready(function () {
   })
   // Initialize the editor with the div id and path to Popcorn Editor.
   PopcornEditor.init('editor', '/static/popcorn/PopcornEditor/editor.html');
+  PopcornEditor.listen('save', function (message) {
+    console.log(message);
+    $.post($('#editor').data('save'), {
+      data: JSON.stringify(message.data),
+      slug: $('#editor').data('slug'),
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val()
+    })
+    .done(function (response) {
+      alert('Event is saved and processing with PopcornEdit.id = '
+           + response.id);
```

Comment:
Oh, I was thinking we'll stick with an `alert` but a mortal don't understand what `PopcordEdit.id = 123` means. 
How about
`alert("Your edit has been saved and sent it for transcoding. Once it completes, which can take several minutes, we'll automatically update the event to use your latest video edits.")`